src/main/webapp/view/video/video.js

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 import Synchronizer from '../../synchronizer.js';
+import {millisecondsToSeconds} from '../../timestamps.js';
```

Comment:
These are on separate lines because being on the same line the line width is 81, max linter accepts is 80.